src/shared/components/ResultsTable/ResultsTable.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Table } from 'antd';
+import './ResultTable.less';
+import moment from 'moment';
+import Search from 'antd/lib/input/Search';
```

Comment:
That's a weird import, isn't it?

I would normally do:
```suggestion
import { Input } from 'antd';

const { Search } = Input;
```